views/home.tmpl

```diff
@@ -245,7 +245,7 @@
                                 </span>
                             </div>
                             <div class="fs12 lh1rem text-black-50">
-                                <span data-target="homepage.targetPct">{{printf "%.2f" .PoolInfo.PercentTarget}}</span> % of target&nbsp;<span>{{intComma .PoolInfo.Target}}</span>
+                                <span data-target="homepage.targetPct">{{printf "%+.2f" .PoolInfo.PercentTarget}}</span> % of target&nbsp;<span>{{intComma .PoolInfo.Target}}</span>
```

Comment:
I think this is fine as is, so don't feel the need to change anything, but as another option, instead of `+/-x% of target 40960`, it could instead say

`x% over target` or `x% under target`

which I think reads better and is more compact.